ezgui/examples/demo.rs

```diff
@@ -193,7 +193,7 @@ impl GUI for App {
         if self.controls.is_checked("paused") {
             EventLoopMode::InputOnly
         } else {
-            EventLoopMode::Animation
+            EventLoopMode::Animation("update".parse().unwrap())
```

Comment:
Rust string APIs are confusing. I think `"foo".to_string()` is more canonical, and that's what's used elsewhere in the codebase

Reply:
Good catch! Thanks!